app-modules/form/src/Filament/Blocks/EducatableEmailFormFieldBlock.php

```diff
@@ -78,11 +85,19 @@ public static function getFormKitSchema(SubmissibleField $field): array
         ];
     }
 
+    /**
+     * @return array<string>
+     */
     public static function getValidationRules(SubmissibleField $field): array
     {
         return ['string', 'email', 'max:255'];
     }
 
+    /**
+     * @param mixed $response
+     *
+     * @return array<string, string>
```

Comment:
This return type is not fully correct as some of these values can be something other than `string` like `null` for example.